contracts/utils/GatewayCaller.sol

```diff
@@ -38,14 +38,21 @@ contract GatewayCaller is IGatewayCaller {
         uint256 gasValue
     ) external payable override {
         if (gasValue > 0) {
+            uint256 executionGasLimit = 0;
+            bool estimateOnChain = false;
```

Comment:
```suggestion
            // Gas for the ITS msg must be estimated off-chain
            bool estimateOnChain = false;
            // No need to set the gas limit since it's not being estimated on-chain
            uint256 executionGasLimit = 0;
```
explain the reasoning for the configuration